packages/backend/server/src/data/migrations/1712068777394-prompts.ts

```diff
@@ -0,0 +1,33 @@
+import { PrismaClient } from '@prisma/client';
+
+import { prompts } from './utils/prompts';
+
+export class Prompts1712068777394 {
+  // do the migration
+  static async up(db: PrismaClient) {
+    await db.$transaction(async tx => {
+      await Promise.all(
+        prompts.map(prompt =>
+          tx.aiPrompt.create({
```

Comment:
createMany

Reply:
`createMany` not support craete with connect yet